packages/components/src/MultiSelect/subcomponents/MultiSelectOptionField/MultiSelectOptionField.module.scss

```diff
@@ -0,0 +1,69 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.multiSelectOptionField {
+  display: flex;
+  align-items: flex-start;
+  gap: $spacing-6;
+  padding: $spacing-6;
+  border-radius: $border-solid-border-radius;
+  position: relative;
+
+  &:hover,
+  &:focus-within {
+    background: $color-blue-100;
+
+    .checkbox {
+      border-color: $color-gray-600;
+      background-color: $color-gray-200;
+
+      &.checked {
+        background-color: $color-gray-600;
+      }
+    }
+  }
+
+  &:focus-within {
+    &::after {
+      width: calc(100% + 2px);
+      height: calc(100% + 2px);
+      content: "";
+      pointer-events: none;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      border-radius: $border-focus-ring-border-radius;
+      border-width: $border-focus-ring-border-width;
+      border-style: $border-focus-ring-border-style;
+      border-color: $color-blue-500;
+    }
```

Comment:
Looks like something we need a utility for, put a ticket in the backlog to create one.

Reply:
As a MultiSelect thing or just a general style util for all - seems pretty handy to have for multiple things

Reply:
Yeah for all of our components

Reply:
https://cultureamp.atlassian.net/browse/KDS-1822